track_analyzing/track_analyzer/cmd_cpp_track.cpp

```diff
@@ -0,0 +1,31 @@
+#include "track_analyzing/track.hpp"
+#include "track_analyzing/utils.hpp"
+
+#include "map/routing_helpers.hpp"
+
+#include "storage/storage.hpp"
+
+using namespace routing;
+using namespace std;
+
+namespace track_analyzing
+{
+void CmdCppTrack(string const & trackFile, string const & mwmName, string const & user,
+                 size_t trackIdx)
+{
+  storage::Storage storage;
+  auto const numMwmIds = CreateNumMwmIds(storage);
+  MwmToMatchedTracks mwmToMatchedTracks;
+  ReadTracks(numMwmIds, trackFile, mwmToMatchedTracks);
+
+  MatchedTrack const & track =
+      GetMatchedTrack(mwmToMatchedTracks, *numMwmIds, mwmName, user, trackIdx);
+
+  cout.precision(8);
```

Comment:
How about to restore old precision settings for cout before return from the function?